frontends/web/src/components/dialog/dialog.module.css

```diff
@@ -26,6 +26,15 @@
     max-height: 100vh;
     overflow: auto;
 }
+/* guard dialog and wait-dialog from view styles */
+@media (min-width: 1200px) {
+    .header .title {
+        --size-subheader: 16px;
+    }
+    .modal .contentContainer p {
+        --size-default: 16px;
+    }
```

Comment:
More of a design question, do we wanna keep what we have before? If so I think we should set the --size-default to be `14px`.

Original (14px):
<img width="451" alt="Screen Shot 2022-12-26 at 09 24 53" src="https://user-images.githubusercontent.com/28676406/209525384-32c03d0b-5a13-4c1d-9bdd-8509a32f32c3.png">


This PR (16px):
<img width="447" alt="Screen Shot 2022-12-26 at 09 28 59" src="https://user-images.githubusercontent.com/28676406/209525375-d83a771a-bb33-48a8-a736-4212977dcda2.png">

I think 14px looks good, but we can reduce the top margin/spacing of the "You will be presented..." text. 16px looks slightly too big IMO. 🙂  But good catch regardless.